legend-engine-executionPlan-execution/src/main/java/org/finos/legend/engine/plan/execution/PlanExecutor.java

```diff
@@ -285,7 +286,7 @@ public Result executeWithArgs(ExecuteArgs executeArgs)
         }
     }
 
-    private void setUpState(SingleExecutionPlan singleExecutionPlan, ExecutionState state, MutableList<CommonProfile> profiles, String user)
+    public static void setUpState(SingleExecutionPlan singleExecutionPlan, ExecutionState state, MutableList<CommonProfile> profiles, String user)
```

Comment:
Can we keep this private and make sure the test executes properly so that it goes thru this code the same way production does it?